src/components/button/button.tsx

```diff
@@ -77,7 +77,8 @@ export class Button
   @Prop() readonly size: "large" | "normal" | "small" = "normal";
 
   @Listen("click", { capture: true })
-  private handleClick(event: UIEvent) {
+  handleClick(event: UIEvent) {
+    event.preventDefault();
```

Comment:
Why do we need to prevent the default behavior?

Reply:
I include the preventDefault to prevent unexpected behavior... Do you suggest to remove that line?

Reply:
removed! Check 09ae8dc